pyinfra/operations/selinux.py

```diff
@@ -0,0 +1,177 @@
+"""
+Provides operations to set SELinux file contexts, booleans and port types.
+"""
+from pyinfra import host
+from pyinfra.api import QuoteString, StringCommand, operation
+from pyinfra.facts.selinux import FileContext, FileContextMapping, SEBoolean, SEPort
+
+
+@operation(
+    pipeline_facts={"seboolean": "boolean"},
+)
+def se_boolean(boolean, value, persistent=False):
```

Comment:
```suggestion
def boolean(boolean, value, persistent=False):
```

Consistent with the `file_context` operations I think `selinux.boolean` is suitable.

Reply:
changed.